standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -425,12 +425,12 @@ public String getZkUrl() {
     @Synchronized
     public void close() throws Exception {
         if (isInProcSegmentStore) {
-            for (ServiceStarter starter : this.nodeServiceStarter) {
+            for ( ServiceStarter starter : this.nodeServiceStarter ) {
                 starter.shutdown();
             }
         }
         if (isInProcController) {
-            for (ControllerServiceMain controller : this.controllerServers) {
+            for ( ControllerServiceMain controller : this.controllerServers ) {
```

Comment:
and here

Reply:
fixed https://github.com/pravega/pravega/pull/4769/commits/b93967814181a08081b8c3025588ecba5c6331e9